systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -49,5 +49,12 @@ public static void waitTillContinuousClientsFinish(String producerName, String c
             () -> kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(producerName).get().getStatus().getSucceeded().equals(1) &&
                     kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(consumerName).get().getStatus().getSucceeded().equals(1));
     }
+
+    public static void waitTillClientFinish(String jobName, String namespace, int messageCount) {
```

Comment:
```suggestion
    public static void waitForClientSuccess(String jobName, String namespace, int messageCount) {
```
Just to match syntax of the calling function

Also it only passes on success, not finish